File: code/d7183.pas
program Bai183;
var n,i, dem: integer;
    s :longint;
  begin
  readln(n);
  s := 1;
  for i := 1 to n do s := s*i;
   dem := 0;
  while s mod 10 = 0 do 
    begin
    dem := dem + 1;
    s := s div 10;
    end;
  write(dem);
  end.

File: code/d7183.cpp
#include <iostream>
using namespace std;

int main() 
{
    int i,n,s,dem;
    cin>>n;
    s=1;dem=0;
    for (i=2; i<=n; i++)
        s=s*i;
    while (s%10==0){
        dem++;
        s=s/10;
    }
    cout<<dem;
    return 0;
}